tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Ricci.cpp

```diff
@@ -0,0 +1,134 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Ricci.hpp"
+#include "tests/Unit/PointwiseFunctions/GeneralRelativity/GrTestHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+void test_1d_spatial_ricci(const DataVector& used_for_size) {
+  const size_t spatial_dim = 1;
```

Comment:
Make all these `constexpr` instead of just const

Reply:
Is constexpr preferred over const for all variables that are const?

Reply:
constexpr is preferred for all variables that are constant expressions (known at or can be calculated at compile time). Anything that allocates memory, such as DataVector, unordered_map, etc. is not constexpr (yet)